pubspec.lock

```diff
@@ -138,7 +138,7 @@ packages:
       name: markdown
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.11.2"
+    version: "0.11.3"
```

Comment:
Just want to confirm, that we're not using new API from `0.11.3`? (If so, we'll also need to bump the pubspec)